### DISCUSSION TITLE
Cookie Configuration when using cookies

### DISCUSSION THREAD

**Discussion by Alik2015** [simpleidserver/SimpleIdServer#637] [Ideas]
When using cookie auth on a asp.net core server app the is there a way to rename the cookie?  The cookie is called AspNetCore.Cookies by default.

I know this can be configured in program.cs of the IdServer manually using builder.Services.ConfigureExternalCookie and changing the name but I am wondering if it can just be set in appsettings.  Looking through the implementation it does seem that most cookies options can be centrally set.

**Comment by simpleidserver** [comment]
Indeed, with the current version, it is not easy to override the cookie's configuration.

I have made some modifications in the master branch to provide more flexibility to SimpleIdServer users. Users can now edit various options of the cookie, such as its name.

The AddSIDIdentityServer function now accepts a new parameter, `cookie`. This callback is utilized to modify different properties of `CookieAuthenticationOptions`.

Additionally, I have introduced two new parameters in the appsettings.json file:

* AuthCookieNamePrefix: the name of the authentication cookie.
* SessionCookieNamePrefix: the name of the session cookie.
